slicing.py

```diff
@@ -0,0 +1,13 @@
+x = input("what is the string")
+y = len(x)
+x1 = x[1:-1]
+print(x1)
+
+x2 = x[2:]
+print(x2)
+
+x3 = x[:-2]
+print(x3)
+
+x4 = x[1::1]
```

Comment:
Program isn't user friendly. I can't understand what each output means. Please correct according to the example given.